src/createServer.js

```diff
@@ -1,11 +1,29 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { usersRouter } = require('./api/users.router.js');
+const { expensesRouter } = require('./api/expenses.router.js');
+const { resetUsers } = require('./services/users.service.js');
+const { resetExpenses } = require('./services/expenses.service.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  resetUsers();
+  resetExpenses();
```

Comment:
The `resetUsers` and `resetExpenses` functions are called during server initialization. Ensure that these functions are intended to reset data each time the server starts, as this might not be suitable for a production environment where data persistence is required.